src/Resources/views/standard_layout.html.twig

```diff
@@ -56,28 +56,10 @@ file that was distributed with this source code.
             {% endblock %}
 
             {% block sonata_javascript_pool %}
-                {% for javascript in sonata_admin.adminPool.getOption('javascripts', []) %}
-                    <script src="{{ asset(javascript) }}"></script>
+                {% for name, path in sonata_admin_webpack_entries %}
+                    {{ encore_entry_script_tags(name, null, name) }}
                 {% endfor %}
             {% endblock %}
-
-            {# localize moment #}
-            {% set localeForMoment = canonicalize_locale_for_moment() %}
```

Comment:
Aren't the issues fixed at #4878 back with these changes?
BTW, if the Twig functions added in that PR aren't required anymore, they should be removed I guess.

Reply:
Yea you are right this can be removed. 👍

Reply:
@silasjoisten, could you please confirm if the issues fixed at #4878 aren't present in this PR?

Reply:
@phansys i think there is a need to convert the mapping from this #4878 in javascript. I will update that asap.